extractor/src/main/java/org/schabi/newpipe/extractor/InfoItem.java

```diff
@@ -55,12 +56,12 @@ public String getName() {
         return name;
     }
 
-    public void setThumbnailUrl(final String thumbnailUrl) {
-        this.thumbnailUrl = thumbnailUrl;
+    public void setThumbnails(final List<Image> thumbnails) {
+        this.thumbnails = thumbnails;
     }
 
-    public String getThumbnailUrl() {
-        return thumbnailUrl;
+    public List<Image> getThumbnails() {
+        return thumbnails;
     }
```

Comment:
Please annotate these methods as either `@Nullable` or `@NonNull`. From what I've seen so far, they are likely `@NonNull` and empty if no images were found.

Reply:
Done, annotations added on all `Extractor`s, `InfoItem`s, `InfoItemExtractor`s, `InfoItemsCollector`s and `Info`s.